.travis.yml

```diff
@@ -12,6 +12,8 @@ script:
 - flake8 src
 - cd docs; make html; cd ..;
 - touch ./docs/_build/html/.nojekyll
+- if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then bash ./scripts/validate-version-bump.sh; fi
```

Comment:
why not == "true"?

Reply:
:shrug: I literally copied this from the travis docs:
https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions

Reply:
That's stupid enough that it might be worth a comment

Reply:
:fire: